package.json

```diff
@@ -1,22 +1,33 @@
 {
   "name": "manim-notebook",
   "displayName": "Manim Notebook",
-  "description": "Replicates Grant Sanderson's interactive workflow for manim in VSCode",
+  "description": "ManimGL with interactive previewing to easily visualize and share the beauty of mathematics and related fields.",
```

Comment:
Sounds good. Is there a way to link to Grant's [youtube video](https://www.youtube.com/watch?v=rbu7Zu5X1zI) from the description? It would be nice to emphasize that this simulates Grant's existing workflow for those that may not know of his explanatory video

Reply:
What about adding that information to the Readme? I've added a TODO point in #91.

While originally we replicated Grant's workflow, I think we now have a ton more features and a direct VSCode integration, which is why I'm a bit reluctant of including the link directly in the description. And I think this meta field should not include links anyways. That's why I favor for the Readme solution ;)